docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -5,110 +5,279 @@ description: "A guide to creating backup of a running Zeebe cluster."
+The backup is taken asynchronously in the background while Zeebe is processing.
+Thus, the backups can be taken with minimal impact on normal processing.
+The backups are stored externally and can be used to restore a cluster in case of failures that leads to full data loss or data corruption.
+
+Zeebe provides a Rest API to create backups, and query and manage existing backups.
+The API is accessible via the management port of the gateway.
+
+## Prerequisites
 
-A backup of a Zeebe cluster consists of a consistent snapshot of all partitions. The backup is stored externally. The backup is taken asynchronously in the background while Zeebe is processing. Thus, the backups can be taken with minimal impact on normal processing.
+To use the backup feature in Zeebe, the following configurations must be provided.
 
-Backup of a running Zeebe cluster can be taken using the following rest APIs. The API is accessible via the management port of the gateway.
+- Enable backups by setting the flag `ZEEBE_BROKER_EXPERIMENTAL_FEATURES_ENABLEBACKUP` to `true`.
+- The backup management API is available via management port of the gateway. Ensure the configuration `MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE` include "backups". Set `MANAGEMENT_ENDPOINTS_BACKUPS_ENABLED` to `true`.
+- Backup is stored in an external storage. This must be configured before starting the Zeebe cluster.
+
+Configure backup store in the configuration file as follows.
+
+```
+zeebe:
+  broker:
+    data:
+      backup:
+        store: S3
+          s3:
+            bucketName:
+            basePath:
+            region:
+            endpoint:
+            accessKey:
+            secretKey:
+```
+
+Alternatively, you can configure backup store using environment variables:
 
-- [Trigger backup](#trigger-backup)
-- [Monitor backup](#monitor-backup)
+- `ZEEBE_BROKER_DATA_BACKUP_STORE` - Specify which storage to use as the backup storage. Currently, only S3 is supported. You can use any S3-compatible storage.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_BUCKETNAME` - The backup will be stored in this bucket. The bucket must already exist.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_BASEPATH` - If the bucket is shared with other Zeebe clusters, then a unique basePath must be configured.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_ENDPOINT` - If no endpoint is provided, it will be determined based on the configured region.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_REGION` - If no region is provided, it will be determined [from the environment](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html#automatically-determine-the-aws-region-from-the-environment).
+- `ZEEBE_BROKER_DATA_BACKUP_S3_ACCESSKEY` - If either `accessKey` or `secretKey` is not provided, the credentials will be determined [from the environment](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html#credentials-chain).
+- `ZEEBE_BROKER_DATA_BACKUP_S3_SECRETKEY`
```

Comment:
:question: No docs? :sweat_smile: I guess it's sort of bundled with the above? :shrug: